.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDatePeriodParser.cs

```diff
@@ -742,18 +757,39 @@ private DateTimeResolutionResult ParseDayToDay(string text, DateObject reference
                     beginDay = this.config.DayOfMonth[dayFrom];
                     endDay = this.config.DayOfMonth[dayTo];
                 }
+                else if (match.Groups["half"].Success)
+                {
+                    var er = this.config.DurationExtractor.Extract(match.Groups["half"].Value, referenceDate);
+                    var pr = this.config.DurationParser.Parse(er[0], referenceDate);
+
+                    var numberStr = pr.TimexStr.Substring(pr.TimexStr.IndexOf('P') + 1, pr.TimexStr.IndexOf('D') - 1);
```

Comment:
This should be a method in a timex class that parses it for number, no?

Reply:
moved to TimexUtility